### PR TITLE
FAQ Link Accessibility

### PR DESCRIPTION
**Pull request by rufaida99-k**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
  
  

---

### Description

The FAQ page had a link that was not readable according to the WCAG guidelines so I added some code to change that. 
The files I changed were:
1. frontend/assets/css/tailwind.css
2. frontend/pages/help/faq.vue

Showing a before after picture below to show the difference:
BEFORE:
image

AFTER:
image

### Related issue
#91

### CODE REVIEW

File: frontend/pages/help/faq.vue [hunk 1]
```diff
@@ -30,7 +30,7 @@
       activist uses Matrix for communications. You're welcome to
       <a
         href="https://matrix.to/#/#activist_community:matrix.org"
-        class="text-light-cta-orange dark:text-dark-cta-orange hover:underline focus-brand"
+        class="link-text"
```
Comment (andrewtavis) [inline frontend/pages/help/faq.vue:33]: This should maintain `focus-brand` :)

**Review by andrewtavis — CHANGES_REQUESTED** [review]
Thanks, @rufaida99-k! 🙏🚀 Just have some minor questions on this:

- I don't really understand what's going on with `backend/manage.py`. Can you remove the change to that? It looks like the contents are being deleted 🤔
- Could you go through and replace the link classes on other instances of external links like on the supporters pages, about page and others? I think searching for `hover:underline` should give you most instances of external links 😊
- As I said in [a comment](https://github.com/activist-org/activist/pull/92/files#r1157854912), `focus-brand` should be maintained on the class for external links :) :) The end class should be `"focus-brand link-text"`

Happy to merge when these minor changes go through! Thanks again!

**Comment by rufaida99-k** [comment]
@andrewtavis got it, making the changes on a new branch. 
I didn't realize the manage.py files even got changed, will make sure that doesn't happen again. 
I'll be sure to make the change "focus-brand link-text" instead of "link-text".
Just one quick follow up:
There are elements like "items-center text-light-cta-orange dark:text-dark-cta-orange hover:underline focus-brand" when I search for "hover:underline", do I maintain the leading "items-center" or replace it altogether with  "focus-brand link-text"?
Also thank you so much for the feedback and your patience :)

**Comment by andrewtavis** [comment]
In the future you can just send the commits to this branch, @rufaida99-k 😊 And no need to thank me for feedback and patience! Thank you for your contributions! Will close this one and check the other :)

**Pull request closed without merging**